lib/src/showcase.dart

```diff
@@ -26,11 +26,10 @@ import 'dart:ui';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
-import 'enum.dart';
+import '../showcaseview.dart';
```

Comment:
any reason for this import?

Reply:
No particular reason for this import; it just reduces the number of other imports needed.

Removing this and adding individual file imports.